src/common/models/Projection.ts

```diff
@@ -491,6 +516,42 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityByCases(): Array<number | null> {
+    const caseDensityByCases = [];
+    const totalPopulation = this.totalPopulation;
+    const totalPopulationAboveZero = totalPopulation > 0;
```

Comment:
super nit: I think this calculation (`totalPopulation > 0`) is so trivial that storing it to a variable probably decreases readability a tiny bit with no particular benefit.  The code would be simpler if you omitted the variable and changed the below check to just:

```
if (totalPopulation === 0 || weeklyAverage === null) { ...
```

But this is very stylistic, so feel free to leave as-is